packages/ember-runtime/lib/mixins/sortable.js

```diff
@@ -122,6 +124,11 @@ export default Mixin.create(MutableEnumerable, {
   */
   sortFunction: compare,
 
+  init() {
+    Ember.deprecate(sortableMixinDeprecation, this.isGenerated);
+    this._super.apply(this, arguments);
```

Comment:
Can you tweak to `this._super(...arguments)`, and move the super call before the deprecation?